src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  margin-right: 0;
+  width: auto;
+  font-family: Roboto, sans-serif;
+}
+
+.logo {
+  display: flex;
+  text-decoration: none;
+  margin: 10px 0;
```

Comment:
unnecessary here, you have `align-items: center` on header, so the position of logo would be right